proto/attest/attest.proto

```diff
@@ -12,57 +12,57 @@ option java_package = "org.lfedge.eve.attest";
 // This is the request payload for POST /api/v2/edgeDevice/id/<uuid>/attest
 // The message is assumed to be protected by signing envelope
 message ZAttestReq {
-  ZAttestReqType reqType = 1;  //type of the request
-  ZAttestQuote quote = 2;      //attestation quote msg
-  repeated org.lfedge.eve.certs.ZCert certs = 3;    //X509 certs in .PEM format, signed by device certificate
-  AttestStorageKeys storage_keys = 4;    //encrypted secrets to be saved by the Controller, like encrypted keys for the volume storage vaults
+  ZAttestReqType reqType = 1;                    // type of the request
+  ZAttestQuote quote = 2;                        // attestation quote msg
+  repeated org.lfedge.eve.certs.ZCert certs = 3; // X509 certs in .PEM format, signed by device certificate
+  AttestStorageKeys storage_keys = 4;            // encrypted secrets to be saved by the Controller, like encrypted keys for the volume storage vaults
```

Comment:
Shouldn't this field also have "Storage" removed?

Reply:
done.